CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 # The sequence matters: later patterns take precedence.
 
 # FILES  OWNERS
-*        @usnistgov/opensource-team
+*        @usnistgov/wendellpiez
```

Comment:
@wendellpiez - please review your write permissions to the repository. The CODEOWNERS file shows the following error: "Unknown owner on line 9: make sure the team @usnistgov/wendellpiez exists and has write access to the repository"